code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -246,7 +246,7 @@
 					targeted_organ = H.random_organ_by_process("liver")
 				if (targeted_organ && targeted_organ.is_usable())
 					if (targeted_organ.nature !=MODIFICATION_SILICON) // If randomly chosen organ is prothestic, no damage.
-						targeted_organ.damage += rand (5,10) // How much damage is dealt to each organ. Please adjust for balance
+						targeted_organ.damage += rand (2,8) // How much damage is dealt to each organ. Please adjust for balance
```

Comment:
```suggestion
						targeted_organ.damage += rand (2,4) // How much damage is dealt to each organ. Please adjust for balance
```

Reply:
Woah woah I was trying to be moderate and test the waters, while still leaving it lethal, is this really what you want?